huxley/api/tests/test_delegate.py

```diff
@@ -200,3 +202,283 @@ def test_superuser(self):
             "email" : unicode(self.params['email']),
             "summary" : unicode(self.params['summary']),}
         )
+
+
+class DelegateListGetTestCase(tests.ListAPITestCase):
+    url_name = 'api:delegate_list'
+
+    def setUp(self):
+        self.user = models.new_user(username='regular', password='user')
+        self.school = models.new_school(user=self.user)
+        self.country = models.new_country()
+        self.committee1 = models.new_committee()
+        self.committee2 = models.new_committee()
+        self.assignment1 = models.new_assignment(
+            committee=self.committee1,
+            country=self.country,
+            school=self.school,
+        )
+        self.assignment2 = models.new_assignment(
+            committee=self.committee2,
+            country=self.country,
+            school=self.school,
+        )
+        self.delegate1 = models.new_delegate(
+            assignment=self.assignment1,
+        )
+        self.delegate2 = models.new_delegate(
```

Comment:
It'd help to have a third delegate who's from a different school to make sure the filtering works.